src/components/actionBar/actionBar/ActionBar.tsx

```diff
@@ -87,11 +172,20 @@ const ActionBar: React.FC<IActionBarProps> = ({
                     <Grid item xs={12} sm>
                         <Autocomplete
                             fullWidth
-                            options={locationOptions.filter((loc) => {
-                                return loc != stopLocation;
-                            })}
+                            options={
+                                stopLocation
```

Comment:
good to move the logic outside and assign the values  to variable and use it here..

Reply:
Logic moved to a function that filters and returns required options:https://github.com/nijin-26/bus-ticket-booking-react/pull/24/commits/e0b34ca886845101dae01915eabdf2f0bdde1c75